src/js/poster-image.js

```diff
@@ -46,12 +46,16 @@ class PosterImage extends ClickableComponent {
    *         The element that gets created.
    */
   createEl() {
-    const el = Dom.createEl('div', {
+    const crossOrigin = this.player_.options_.crossOrigin || this.player_.options_.crossorigin || null;
```

Comment:
Should we use the `crossOrigin` getter?
```suggestion
    const crossOrigin = this.player_.crossOrigin() ?? null;
```

Reply:
The poster can be set up before the getter will return a correct value, as it's dependent on the tech

Reply:
Ah, that's right. However, we should probably still use the getter first because otherwise, the value may be out of sync with the rest of the system.